.test-conda-env-py3.yml

```diff
@@ -10,4 +10,4 @@ dependencies:
 - ocl-icd
 - pocl
 - mako
-- pybind11
+- nanobind
```

Comment:
Is it a good idea to install `nanobind` from conda? I wasn't sure if just to remove `pybind11` from here or replace it..

Reply:
AFAIK, the `nanobind` package itself does not need to be compiled (see https://github.com/conda-forge/nanobind-feedstock/blob/main/recipe/meta.yaml), so it's probably fine to use pypi and not conda for it.

Reply:
Fair enough, I went ahead and just removed both `pybind11` and `nanobind` from the conda install.